### PR TITLE
Improved Great People, Part Deux

### PR DESCRIPTION
**Pull request by etiennefd**
Some mistakes corrected and many great people added (and a few removed)

Possibly controversial: I moved Gabriele Trovato (Rhye) from the Artist to the Engineer category because, well, he's an [actual engineer](http://rhye.civfanatics.net/gabu/) (and another game designer, Shigeru Miyamoto, is in the engineer category for Japan)

~ steb

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -666,15 +679,16 @@ def setup():
 		"Yosa Buson", # 18th
 		iIndustrial,
 		"Katsushika Hokusai", # 19th
-		"Utagawa Hiroshige", # 
+		"Utagawa Hiroshige", # 19th
+		"fHiguchi Ichiyou", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Japanese names in the mod follow Western conventions (first name, last name), so it should be `fIchiyou Higuchi`.

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -712,12 +725,13 @@ def setup():
 		iIndustrial,
 		"Sakamoto Ryouma", # 19th
 		"Oukubo Toshimichi", # 19th
+		"Fukuzawa Yukichi", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Same here, Yukichi Fukuzawa.

File: Assets/Python/GreatPeople.py [hunk 3]
```diff
@@ -1185,51 +1204,61 @@ def setup():
 		iRenaissance,
 		"Charles Le Brun", # 17th
 		"Jean-Baptiste Lully", # 17th
-		"Jean-Antoine Watteau", # 17th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:1188]: Why did you remove him?

File: Assets/Python/GreatPeople.py [hunk 4]
```diff
@@ -1185,51 +1204,61 @@ def setup():
...
 		u"Molière", # 17th
 		"Voltaire", # 18th
+		u"fÉlisabeth Vigée Le Brun", # 18th
 		iIndustrial,
 		"Victor Hugo", # 19th
 		"Claude Monet", # 19th
 		"Henri Matisse", # 19th
 		"Claude Debussy", # 19th
 		"fGeorge Sand", # 19th
-		"Alexandre Dumas", # 19th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Same here, why remove him?

File: Assets/Python/GreatPeople.py [hunk 5]
```diff
@@ -1185,51 +1204,61 @@ def setup():
...
 		"fMarie-Sophie Germain", # 19th
 		"fMarie Curie", # 19th
 		iGlobal,
 		"Antoine Henri Becquerel", # 20th
+		"Jacques Monod", # 20th
 	],
 	iGreatMerchant : [
 		"Jacques Cartier", # 16th
 		"Samuel de Champlain", # 17th
-		u"fThérèse de Couagne", # 18th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Why did you remove her?

File: Assets/Python/GreatPeople.py [hunk 6]
```diff
@@ -1265,11 +1293,18 @@ def setup():
 		"Charles V", # 14th
 		"fJeanne d'Arc", # 15th
 		iRenaissance,
+		u"Louis de Bouron-Condé", # 17th
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: I can't find who that is referring to?

**Review by dguenms — CHANGES_REQUESTED** [review]
This is great, exactly what the GP names module should be about. I agree with all suggestions/corrections. Only a couple of questions about removed names and tiny corrections for Japanese names.

File: Assets/Python/GreatPeople.py (hunk 3 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py:1188]: Basically I replaced him with Élizabeth Vigée Le Brun who seems to me more noteworthy, and I felt he could be removed since the French GA list is already quite long. There isn't much more reason than that, let me know if you prefer him in

File: Assets/Python/GreatPeople.py (hunk 4 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: See above; I replaced Dumas with Balzac

File: Assets/Python/GreatPeople.py (hunk 5 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: I changed my mind, she should be in

File: Assets/Python/GreatPeople.py (hunk 6 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: There's a typo: it's [Louis de Bourbon-Condé](https://fr.wikipedia.org/wiki/Louis_II_de_Bourbon-Cond%C3%A9)

File: Assets/Python/GreatPeople.py (hunk 1 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py]: Oops, you're right, will change

**Comment by etiennefd** [comment]
I made the requested changes. And you know what, I put Watteau, Dumas and de Couagne back in.

File: Assets/Python/GreatPeople.py (hunk 3 above)
Comment (dguenms) [inline Assets/Python/GreatPeople.py]: Looking into him a bit, it seems justified.

**Review by dguenms — APPROVED** [review]
